daemon/cluster/executor/container/container.go

```diff
@@ -77,8 +77,18 @@ func (c *containerConfig) name() string {
 		return c.task.Annotations.Name
 	}
 
-	// fallback to service.slot.id.
-	return strings.Join([]string{c.task.ServiceAnnotations.Name, fmt.Sprint(c.task.Slot), c.task.ID}, ".")
+	// fallback to service.slot.id
+	return fmt.Sprintf("%s.%s", c.shortName(), c.task.ID)
```

Comment:
Sorry, I think my comment was misleading.  Wont this use a name of `service.slot.taskid` ?